### PR TITLE
Re forked since previous merge conflict

### PR DESCRIPTION
**Pull request by jacgit18**
will re attempt other question and try others

### CODE REVIEW

File: joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209BinaryTree..js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+var minSubArrayLen = function(target, nums) {
```
Comment (MelissaPastore) [inline joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209BinaryTree..js]: It doesn't seem like this solution passes the second test case. Did you test your code before submitting?

File: joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+// Problem Statement #
```
Comment (MelissaPastore) [inline joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209.js]: Would be great to see a pseudocode overview and/or some detailed comments on your approach.

File: joshuaCarpentier (copy)/assignments/slidingwindow/lc53/lc53.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+function max_subarray_size_k(k, arr) {
```
Comment (MelissaPastore) [inline joshuaCarpentier (copy)/assignments/slidingwindow/lc53/lc53.js]: Would be great to see some pseudocode re: your approach and/or detailed comments.

File: joshuaCarpentier/assignments/twopointers/w1d2/lc1.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
+// Problem Statement #
```
Comment (MelissaPastore) [inline joshuaCarpentier/assignments/twopointers/w1d2/lc1.js]: Nice pseudocode.

**Review by MelissaPastore — CHANGES_REQUESTED** [review]
It doesn't seem like the solution for lc209 is passing all of the test cases. Did you test your code before submitting it?

File: joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209BinaryTree..js (hunk 1 above)
Comment (jacgit18) [reply joshuaCarpentier (copy)/assignments/slidingwindow/lc209/lc209BinaryTree..js]: I was just testing and observing this example I found it my attempt was on the solution without the binary search

**Pull request closed without merging**